release-tools/repack-debian-tarball.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/sh
+# This script is used to re-pack the "orig" tarball from the Debian package
+# into a suitable upstream release. There are two changes applied: The Debian
+# tarball contains the directory snapd.upstream/ which needs to become
+# snapd-$VERSION. The Debian tarball contains the vendor/ directory which must
+# be removed from one of those.
+#
+# Example usage, using tarball from the archive or from the image ppa:
+#
+# $ wget https://launchpad.net/ubuntu/+archive/primary/+files/snapd_2.31.2.tar.xz 
+# $ wget https://launchpad.net/~snappy-dev/+archive/ubuntu/image/+files/snapd_2.32.1.tar.xz
```

Comment:
The second one is in the PPA, the first one is in the archive. The versions are different because of a skew in how we did releases. This is just a reminder of where they usually are.